src/core/components/form/DatetimeRange.vue

```diff
@@ -82,11 +82,31 @@ export default {
         if (key === 'startHour') {
```

Comment:
Dans ce fichier quand on met a jour la date de fin, on la met a "end of day", je pense qu'il faudrait uniquement mettre a jour l'annee, mois et jour pour garder les secondes & millisecondes

Reply:
tu as raison